api/v1/views/index.py

```diff
@@ -4,10 +4,23 @@
 """
 
 from . import app_views
+from models import storage
+from flask.json import jsonify
 
 
 @app_views.route("/status")
 def status():
-    return {
-        "status": "OK"
-    }
+    return jsonify(status="OK")
```

Comment:
Who's idea was this in the first place? The second form is MUCH better. Good work